.github/workflows/integration.yml

```diff
@@ -279,30 +287,57 @@ jobs:
       - name: run proposals tests
         run: yarn test
         working-directory: a3p-integration
-      - name: collect all core eval scripts
-        # Core eval scripts will be copied under /tmp/core_eval_scripts directory
-        # colons in the parent directory name will be replaced with dashes
+      - name: copy a3p-integration
+        if: (success() || failure())
         run: |
-          find . -type d -path "./a3p-integration/proposals/*/submission" | while read -r dir; do
-            # Get the parent directory name
-            parent_dir=$(basename "$(dirname "$dir")")
-            # Replace colons with dashes in the parent directory name
-            sanitized_parent_dir=${parent_dir//:/-}
-            # Create the destination directory under /tmp/core_eval_scripts if it doesn't exist
-            destination_dir="/tmp/core_eval_scripts/$sanitized_parent_dir"
-            mkdir -p "$destination_dir"
-            # Copy everything from the $dir to the destination directory
-            cp -r "$dir"/* "$destination_dir/"
-            echo "Copied contents of $dir to $destination_dir"
+          # Copy everything from a3p-integration to /tmp/export/a3p-integration
```

Comment:
Hmmmm. While it is impressive that you are performing this feat with both hands tied behind your back (with Bash under YAML), would it be easier to call a Node.js script leveraging `execa` instead?

Reply:
Meh, no strong feelings either way. I kept it like this as an evolution of what was already there, but am equally happy to change it.

Reply:
Okay, let's split the difference, and excise the YAML.  Can you factor out the Bash into a file like `agoric-sdk/scripts/$something.sh`, and call it from the workflow?

Reply:
Done.